dpdata/plugins/deepmd.py

```diff
@@ -69,6 +71,88 @@ def from_labeled_system(self, file_name, type_map=None, **kwargs):
     MultiMode = Format.MultiModes.Directory
 
 
+@Format.register("deepmd/mixed")
```

Comment:
I think it's actually `deepmd/npy/mixed`. We can also have `deepmd/raw/mixed` and `deepmd/hdf5/mixed`.